community.md

```diff
@@ -30,6 +30,10 @@ any issue you find with Presto.
 
 ## External Resources
 
+### Books
+
+* [Presto: The Definitive Guide](./presto-the-definitive-guide.html)
+
```

Comment:
We may also want to add it to the resources page.

Reply:
the resources page is kinda odd .. its only libraries and such .. I added it here after chatting with @electrum a while ago .. but I can add something there as well if you like ..

Reply:
No, that's ok. Let's keep it as is. We can change it or add it later.